packages/cli/src/commands/dev.js

```diff
@@ -51,6 +52,9 @@ export const handler = async ({
   const API_DIR_SRC = getPaths().api.src
   const WEB_DIR_SRC = getPaths().web.src
 
+  // Run tasks like type generate, etc.
+  runPreBuildTasks()
+
```

Comment:
Wouldn't we only want this to run for the api-side?

Reply:
No I don't think so, because `useAuth` also gets typed for example.

Reply:
Ok, I thought the code block was checking for the existence of the api folder, but I see now that it's only checking to see if the side was specified in the command.